test/membrane_mp4/muxer/isom/integration_test.exs

```diff
@@ -15,24 +15,28 @@ defmodule Membrane.MP4.Muxer.ISOM.IntegrationTest do
   defp out_path_for(filename), do: "/tmp/out_#{filename}.mp4"
   defp ref_path_for(filename), do: "test/fixtures/isom/ref_#{filename}.mp4"
 
-  defp perform_test(pid, filename) do
+  defp prepare_test(filename) do
     out_path = out_path_for(filename)
-    ref_path = ref_path_for(filename)
-
     File.rm(out_path)
     on_exit(fn -> File.rm(out_path) end)
+  end
+
+  defp perform_test(pid, filename) do
+    out_path = out_path_for(filename)
+    ref_path = ref_path_for(filename)
 
-    assert Pipeline.play(pid) == :ok
     assert_end_of_stream(pid, :sink, :input)
     refute_sink_buffer(pid, :sink, _buffer, 0)
 
-    assert :ok == Pipeline.stop_and_terminate(pid, blocking?: true)
+    assert :ok == Pipeline.terminate(pid, blocking?: true)
 
     assert_files_equal(out_path, ref_path)
   end
 
   describe "Muxer.ISOM should mux" do
     test "single H264 track" do
+      prepare_test("video")
```

Comment:
Now Pipeline starts automatically after calling `start/start_link`. Therefore, I moved deleting files to a separate function to not delete files when Pipeline is running.

Reply:
Wouldn't it be better to rewrite the tests to use `:tmp_dir`? Just a nitpick though, it can stay like that for now if you don't have the time

Reply:
I belive they are using `/tmp` dir under the hood but you still need to remove files as you might run it multiple times on the same machine